zkevm-circuits/src/evm_circuit/util/constraint_builder.rs

```diff
@@ -656,6 +657,13 @@ impl<'a, F: Field> EVMConstraintBuilder<'a, F> {
         self.opcode_lookup_rlc(opcode, 0.expr());
     }
 
+    #[cfg(feature = "dual_bytecode")]
+    pub(crate) fn opcode_lookup2(&mut self, opcode: Expression<F>, is_code: Expression<F>) {
+        assert_eq!(is_code, 1.expr());
+        self.opcode_lookup_rlc2(opcode, 0.expr());
```

Comment:
`assert_eq!(is_code, 1.expr());` i would be surprise if this line runs ok...   we really want to check `is_code` is a constant expression?  if yes, why not remove the constant redundant parameter from function definition


reading more to understand this.

Reply:
it is historical writing style in old method `opcode_lookup`, call this method to lookup real opcode (inner call `opcode_lookup_rlc`  with is_code = 1 ), but i think can remove it.

Reply:
removed in [a3a88de](https://github.com/scroll-tech/zkevm-circuits/pull/1368/commits/a3a88de0d1ebb267b7ca228a6d0cb80e130d914e) and add comment to explain this helper lookup real opcode.